src/main/java/org/opensearch/knn/index/IndexUtil.java

```diff
@@ -119,6 +121,18 @@ private static Object getFieldMapping(final Map<String, Object> properties, fina
         return currentFieldMapping;
     }
 
+    /**
+     *  This method is used to get the vector data type from field mapping
+     * @param fieldMap field mapping
+     * @return vector data type
+     */
+    private static VectorDataType getVectorDataTypeFromFieldMapping(Map<String, Object> fieldMap) {
```

Comment:
Move all Private Functions to end of the file .

Reply:
done